package.json

```diff
@@ -25,7 +25,7 @@
     "@angular/platform-browser-dynamic": "^5.2.0",
     "@angular/router": "^5.2.0",
     "core-js": "^2.4.1",
-    "ngx-twitter-timeline": "^0.1.4",
+    "ngx-twitter-timeline": "0.1.4",
```

Comment:
Writting a comment about why we are using specific version here, will be better to remind this change. Also in Angular 7 project this can be changed to latest version of ngx-twitter-timeline.